bin.src/allocateNodes.py

```diff
@@ -40,8 +41,8 @@ def main():
     remoteLoginCmd = "/usr/bin/gsissh"
     remoteCopyCmd = "/usr/bin/gsiscp"
 
-    UNKNOWN_PLATFORM_EXIT_CODE = 10
-    MISSING_PBS_CONFIG_EXIT_CODE = 20
+    # UNKNOWN_PLATFORM_EXIT_CODE = 10
+    # MISSING_PBS_CONFIG_EXIT_CODE = 20
```

Comment:
These are intended to be used at a later date when we have severity levels.  I'm looking into this later this month.